cogito/gchatsink.go

```diff
@@ -12,9 +13,10 @@ import (
 
 // GoogleChatSink is an implementation of [Sinker] for the Cogito resource.
 type GoogleChatSink struct {
-	Log     hclog.Logger
-	GitRef  string
-	Request PutRequest
+	Log      hclog.Logger
+	InputDir fs.FS
```

Comment:
Totally unrelated to this PR, but just for my curiosity.

I forgot, but can you remind be why we were using `afero` in `fly_helper`. It was something about read/write and `fs.FS` supporting only read, but how you are using the `fstest` here, I am not sure anymore if `afero` was needed. 

What do we write with fly_helper that we (and by this I mean I :) ) decided for afero?

Reply:
Maybe we write nothing :-D I do not remember :-D

Reply:
I remember the plan for `fly_helper init` command, but for sure this was not the reason. I will have a look again.